collect_app/src/main/java/org/odk/collect/android/tasks/ServerPollingJob.java

```diff
@@ -0,0 +1,207 @@
+                        informAboutNewDownloadedForms(FormDownloadList.getDownloadResultMessage(result));
+                    } else {
+                        for (FormDetails formDetails : newDetectedForms) {
+                            String manifestFileHash = formDetails.getManifestFileHash() != null ? formDetails.getManifestFileHash() : "";
+                            String formVersionHash = FormDownloader.getMd5Hash(formDetails.getHash()) + manifestFileHash;
+                            if (!wasThisNewerFormVersionAlreadyDetected(formVersionHash)) {
+                                updateLastDetectedFormVersionHash(formDetails.getFormID(), formVersionHash);
+                            } else {
+                                newDetectedForms.remove(formDetails);
+                            }
+                        }
+
+                        if (!newDetectedForms.isEmpty()) {
+                            informAboutNewAvailableForms();
+                        }
+                    }
+                }
+                return Result.SUCCESS;
+            } else {
+                return Result.FAILURE;
+            }
+        }
+    }
+
+    public static void schedulePeriodicJob(String selectedOption) {
+        if (selectedOption.equals(Collect.getInstance().getString(R.string.never_value))) {
+            JobManager.instance().cancelAllForTag(TAG);
+            GeneralSharedPreferences.getInstance().reset(POLL_SERVER_IMMEDIATELY_AFTER_RECEIVING_NETWORK);
+        } else {
+            long period = FIFTEEN_MINUTES_PERIOD;
+            if (selectedOption.equals(Collect.getInstance().getString(R.string.every_one_hour_value))) {
+                period = ONE_HOUR_PERIOD;
+            } else if (selectedOption.equals(Collect.getInstance().getString(R.string.every_six_hours_value))) {
+                period = SIX_HOURS_PERIOD;
+            } else if (selectedOption.equals(Collect.getInstance().getString(R.string.every_one_day_value))) {
+                period = ONE_DAY_PERIOD;
+            }
+
+            new JobRequest.Builder(TAG)
+                    .setPeriodic(period, 300000)
+                    .setUpdateCurrent(true)
+                    .build()
+                    .schedule();
+        }
+    }
+
+    private boolean wasThisNewerFormVersionAlreadyDetected(String formVersionHash) {
+        Cursor cursor = new FormsDao().getFormsCursor(LAST_DETECTED_FORM_VERSION_HASH + "=?", new String[]{formVersionHash});
+        return cursor == null || cursor.getCount() > 0;
+    }
+
+    private void informAboutNewAvailableForms() {
+        Intent intent = new Intent(getContext(), FormDownloadList.class);
+        intent.putExtra(DISPLAY_ONLY_UPDATED_FORMS, true);
+        PendingIntent contentIntent = PendingIntent.getActivity(getContext(), 0, intent, PendingIntent.FLAG_UPDATE_CURRENT);
+
+        NotificationCompat.Builder builder = new NotificationCompat.Builder(getContext())
```

Comment:
This looks a little out of place on my Moto. I would remove the Collect icon. It's not what notifications look like normally and it adds to the vertical space. I think what would be nice is a small clipboard instead of the ℹ️ button. 

![screen shot 2018-05-07 at 13 38 03](https://user-images.githubusercontent.com/32369/39723906-15a10f5c-51fc-11e8-81f2-3bd409bb3a43.png)